mrjob/yarnemr.py

```diff
@@ -36,6 +39,16 @@
 
 log = logging.getLogger(__name__)
 
+
+# Since our file cache of cluster info will grow unboundedly as clusters
+# terminate and new clusters start we must cleanup at some point. When the
+# cache is strictly greater than this many days old, we delete the cache
+# and begin again.
+_CLUSTER_FILE_CACHE_TTL = 3
+
+# Location of file
+_CLUSTER_FILE_CACHE_FILE_PATH = '/tmp/mrjob_cluster_file_cache'
```

Comment:
Making this path configurable is better so we can manage the docker mount for it in some random dir.